src/renderer/src/components/exchange/Exchange.tsx

```
@@ -1,4 +1,6 @@
 import { ReactElement, useEffect } from 'react';
+import * as React from 'react';
```

ESLint

[error] 'React' is defined but never used.

(@typescript-eslint/no-unused-vars)